src/utils/constants.ts

```diff
@@ -9,8 +9,7 @@ export const nodesURL: string = baseAPIURL.endsWith('/')
   ? `${baseAPIURL}nodes/`
   : `${baseAPIURL}/nodes/`;
 
-export const isFederationAPI: string =
-  import.meta.env.NB_IS_FEDERATION_API === undefined ? true : import.meta.env.NB_IS_FEDERATION_API;
+export const isFederationAPI: boolean = import.meta.env.NB_IS_FEDERATION_API === 'true';
```

Comment:
I believe this is changing the default behaviour from "undefined -> true" to "undefined -> false". Can you please

1. Confirm this is not the case / otherwise fix
2. Add explicit tests for 
  - env.NB_IS_FEDERATION_API = undefined
  - env.NB_IS_FEDERATION_API = true
  - env.NB_IS_FEDERATION_API = false

so we know for sure and don't have to worry

Reply:
I've updated the fix so the default remains unchanged.
I wasn't able to find a similar workaround that we used in Vue for overwriting the env vars. Here is the section of the Cypress docs that outlines `mount` is frameworks-specific hence why we may not be able to duplicate what we had in Vue: https://docs.cypress.io/api/commands/mount#Creating-a-New-cymount-Command